packages/salesforce-adapter/src/filters/utils.ts

```diff
@@ -525,3 +525,7 @@ export const getChangedAtSingleton = async (
 export const isCustomType = (element: Element): element is ObjectType => (
   isObjectType(element) && ENDS_WITH_CUSTOM_SUFFIX_REGEX.test(apiNameSync(element) ?? '')
 )
+
+export const asListType = (type: TypeElement): ListType => (
```

Comment:
super nit: I find this new naming suggestion consistent with the lodash _.toArray which _feels_ slightly better IMO.
```suggestion
export const toListType = (type: TypeElement): ListType => (
```